test/TestParser.java

```diff
@@ -0,0 +1,144 @@
+import java.io.*;
+import org.junit.After;
+import org.junit.Ignore;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class TestParser {
+   private final String fileName = "test.txt";
+   private final String unicodeString = "Юникодтекст";
+   private final String nonUnicodeString = "Simple text";
+   public void saveStringToFile(String str) {
+      File f = new File( this.fileName );
+      try {
+         f.createNewFile();
+         PrintWriter out = new PrintWriter(f.getAbsoluteFile());
+         try {
+            out.print( str );
+         } finally {
+            out.close();
+         }
+      } catch(IOException e) {
+         throw new RuntimeException(e);
+      }
+   }
+   public String readFromFile() throws IOException {
+      File f = new File( this.fileName );
+      StringBuilder sb = new StringBuilder();
+      if(f.exists() & f.canRead() & f.isFile()) {
+         BufferedReader br = new BufferedReader(new InputStreamReader(new FileInputStream(f),"UTF-8"));
+         String s = "";
+         while((s = br.readLine()) != null)
+            sb.append(s);
+         br.close();
+      }
+      return new String(sb);
+   }
+   @After
+   public void teardownDeleteFile() {
+      File f = new File( this.fileName );
+      if (f.exists() & f.isFile()) {
+         f.delete();
+      }
+   }
+   @Test
+   public void testAccessors() {
+      Parser p = new Parser();
+      p.setFile( new File( this.fileName ) );
+      assertEquals( p.getFile().getName(), this.fileName );
+   }
+   @Test
+   public void testGetContent_Simple() throws IOException {
+      String str = this.nonUnicodeString;
+      saveStringToFile( str );
```

Comment:
saveStringToFile( str ); -> saveStringToFile(str);

no need in extra spacing, here and anywhere else